core/src/main/java/org/kiwix/kiwixmobile/core/utils/SharedPreferenceUtil.kt

```diff
@@ -74,17 +74,17 @@ class SharedPreferenceUtil @Inject constructor(context: Context?) {
       if (storage == null) {
         val defaultStorage = defaultStorage()
         putPrefStorage(defaultStorage)
-        return defaultStorage().also(this@SharedPreferenceUtil::putPrefStorage)
+        return defaultStorage().also(::putPrefStorage)
       } else if (!File(storage).exists()) {
         return defaultStorage()
       }
       return storage
     }
 
   private fun defaultStorage(): String =
-    (getExternalFilesDirs(instance, null)[0])?.let { it.path } ?: instance.filesDir.path
+    getExternalFilesDirs(instance, null)[0]?.let { it.path } ?: instance.filesDir.path
```

Comment:
Here this `let` can be removed to make this line more compact like 
`getExternalFilesDirs(instance, null)[0]?.path ?: instance.filesDir.path`
But I didn't do this as the previous one(the one used in this commit) was more legible and understandable.
Should this `let` be removed to the more concise one?

Reply:
yeah absolutely